staging/nvidia/charts/nvidia-device-plugin/values.yaml

```
@@ -0,0 +1,58 @@
+# Default values for nvidia-device-plugin.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nvidia/k8s-device-plugin
+  tag: 1.0.0-beta
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector:
+  konvoy.mesosphere.com/gpu-provider: NVIDIA
+
+tolerations:
+  # Allow this pod to be rescheduled while the node is in "critical add-ons only" mode.
+  # This, along with the annotation above marks this pod as a critical add-on.
+  - key: CriticalAddonsOnly
+    operator: Exists
+  - key: nvidia.com/gpu
+    operator: Exists
+    effect: NoSchedule
+
+affinity: {}
```

I would recommend adding support for `initContainers` here and now so we don't have to make a PR for it later if needed, and to support more versatility for end-users if they have some other tasks they need to run in init before deploying the addon.

```suggestion
affinity: {}

initContainers: []
```

This will require a change in the `Daemonset` to support this as well.